docs/source/_templates/layout.html

```diff
@@ -1,8 +1,9 @@
 {% extends "!layout.html" %}
 
+<link rel="canonical" href="{{ theme_canonical_url }}{{ pagename }}.html" />
 {% block menu %}
 <div>
-  <a style="color:#F05732" href=http://pytorch.org/docs/stable/>
+  <a style="color:#F05732" href="{{ theme_canonical_url }}{{ pagename }}.html">
```

Comment:
Hmm actually I'm not sure if we need it. Let's say the user click on a link to go somewhere (https://pytorch.org/docs/master/cuda.html#torch.cuda.get_device_name). They won't see be able to see the "you are on unstable pytorch docs" warning, because they're probably somewhere below the top of the page, so...

Reply:
indeed, I totally agree with you.